repos/system_upgrade/common/actors/openssl/opensslconfigscanner/tests/test_opensslconfigscanner.py

```diff
@@ -143,7 +143,6 @@ def fake_producer(*args):
 
 
 @pytest.mark.parametrize(('source_version', 'should_run'), [
-    ('7', False),
     ('8', True),
     ('9', True),
 ])
```

Comment:
same here. no version check anywhere in the actor and always true, so parametrization could be dropped as well